webapi/src/main/resources/knora-ontologies/knora-base.ttl

```diff
@@ -2522,3 +2527,58 @@
                            "Sequence Bounds"@en ;
     rdfs:comment           "Deprecated in favour of :Segment"@en ;
     rdfs:comment           "Indicates the bounds of a sequence, i.e. the start and end point in the containing resource."@en .
+
+
+### http://www.knora.org/ontology/knora-base#TextValueType
+
+:TextValueType a rdf:Alt ;
```

Comment:
knora-base is an owl ontology, therefore it would make sense to solve this also with `owl:oneOf`. Otherwise it sounds like a solution but we can talk about it some more if you like.

Reply:
good point, thanks! Is it correct like this? https://github.com/dasch-swiss/dsp-api/commit/659b576271cc9ea9698be8da17183a2194c90900